spec/models/business_entity_spec.rb

```diff
@@ -29,6 +29,20 @@
     end
   end
 
+  context 'scopes' do
+    let!(:hmcts)   { create(:office, name: 'HMCTS HQ Team ') }
+    let!(:digital) { create(:office, name: 'Digital') }
+    let!(:bristol) { create(:office, name: 'Bristol') }
+
+    describe 'non_digital' do
+      it 'excludes HQ business entities' do
+        # each office gets 2 business_entities by default
+        expect(described_class.count).to eql(6)
+        expect(described_class.exclude_hq_teams.count).to eql(2)
```

Comment:
In here I would test that those 2 BE are the one from the Bristol office. To make the test more robust. Because the underlying scope could change to include some other 2 and the test wouldn't break.